scripts/testing_scripts/update_lib.sh

```diff
@@ -13,18 +13,36 @@ if [ "$1" = kokkos-dev ]; then
     module swap sems-gcc/4.9.3 sems-gcc/6.4.0
     module list
   fi
+  CLANGVER=$(clang --version | grep "clang version" | cut -d " " -f 3)
+  if [[ "${CLANGVER}" = 9.* ]] || [[ "${CLANGVER}" = 10.* ]]; then
+    # Newer gcc needed for c++ standard beyond c++14
+    module swap sems-gcc/5.3.0 sems-gcc/6.4.0
+    module list
+  fi
 fi
 if [ "$1" = kokkos-dev-2 ]; then
   ICPCVER="$(icpc --version | grep icpc | cut -d ' ' -f 3)"
   if [[ "${ICPCVER}" = 17.* ]]; then
     module swap sems-gcc/4.9.3 sems-gcc/6.4.0
     module list
   fi
+  CLANGVER=$(clang --version | grep "clang version" | cut -d " " -f 3)
+  if [[ "${CLANGVER}" = 9.* ]] || [[ "${CLANGVER}" = 10.* ]]; then
+    # Newer gcc needed for c++ standard beyond c++14
+    module swap sems-gcc/5.3.0 sems-gcc/6.4.0
+    module list
+  fi
 fi
-if [ "$1" = sems ]; then
+if [ "$1" = sems ] || [ "$1" = sogpu ]; then
   ICPCVER="$(icpc --version | grep icpc | cut -d ' ' -f 3)"
   if [[ "${ICPCVER}" = 17.* ]]; then
     module swap sems-gcc/4.9.3 sems-gcc/6.4.0
     module list
   fi
+  CLANGVER=$(clang --version | grep "clang version" | cut -d " " -f 3)
+  if [[ "${CLANGVER}" = 9.* ]] || [[ "${CLANGVER}" = 10.* ]]; then
+    # Newer gcc needed for c++ standard beyond c++14
+    module swap sems-gcc/5.3.0 sems-gcc/6.4.0
+    module list
+  fi
```

Comment:
Same as above.

Reply:
Replaced by function in recent push, thanks for feedback